autosklearn/util/backend.py

```diff
@@ -481,9 +479,8 @@ def save_predictions_as_txt(self, predictions, subset, idx, precision, prefix=No
         os.rename(tempname, filepath)
 
     def write_txt_file(self, filepath, data, name):
-        lock_file = filepath + '.lock'
-        with lockfile.LockFile(lock_file):
-            if not os.path.exists(lock_file):
```

Comment:
I think in the way the code was, this `if` statement was always evaluated to `True`, because the `lock_file` is never created by anyone.

Reply:
True. On the other hand, I'm not really sure if we actively still use this function.